### PR TITLE
[10.0] Migration res_currency_print_on_check to 10.0

### PR DESCRIPTION
**Pull request by osvalr**
## Summary
- [x] Change imports, tags
- [x] Remove res.currency records: due there aren't present on [10.0](https://github.com/Vauxoo/odoo/blob/10.0/odoo/addons/base/res/res_currency_data.xml) but they do in [here](https://github.com/Vauxoo/odoo/blob/9.0/openerp/addons/base/res/res_currency_data.xml#L1204) and [here](https://github.com/Vauxoo/odoo/blob/9.0/openerp/addons/base/res/res_currency_data.xml#L952)

### CODE REVIEW

**Comment by osvalr** [comment]
@nhomar this is ready for review

**Comment by moylop260** [comment]
Could you explain a little more why this REM is needed https://github.com/Vauxoo/account-payment/pull/3/commits/9c4e3bf8011775814f860842369d3c4e6eaaa731?

**Comment by osvalr** [comment]
@moylop260 sure, I just updated the description, and the main reason is that they do not exists on newest version of odoo.

**Review by moylop260 — APPROVED** [review]

**Comment by nhomar** [comment]
Can you check if this was not done already in oca/account-payment? if yes please check if it is ok if not please make a proper PR there also.

Regards.

**Comment by osvalr** [comment]
@nhomar @moylop260 res_currency_print_on_check does not exists yet for 10.0, I've created a [task](https://www.vauxoo.com/web?db=vauxoo#id=6475&view_type=form&model=project.task&menu_id=136&action=138) to get it done later, can we proceed to merge this is there is not any other comments?

File: .travis.yml [hunk 1]
```diff
@@ -26,7 +17,7 @@ env:
   - VERSION="10.0" TESTS="0" LINT_CHECK="0"
   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
+  - TESTS="1" ODOO_REPO="vauxoo/odoo"
```
Comment (nhomar) [inline .travis.yml]: you do not need to change this to vauxoo/odoo only the main app repository is the one there.

**Review by nhomar — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (osvalr) [reply .travis.yml]: now i get it

**Pull request merged**